XCUITests/ToolbarTest.swift

```diff
@@ -96,6 +96,28 @@ class ToolbarTests: BaseTestCase {
         XCTAssertEqual(value as? String, "", "The url has not been removed correctly")
     }
 
+    //Check that after scrolling on a page, the URL bar is hidden. Tapping one on the status bar will reveal the URL bar, tapping again on the status will scroll to the top
+    func testRevealToolbarWhenTappingOnStatusbar(){
+        //Workaround when testing on iPhone. If the orientation is in landscape on iPhone the tests will fail.
+        if !iPad() {
+            XCUIDevice.shared.orientation = UIDeviceOrientation.portrait
+        }
+        waitForExistence(app.buttons["HomePanels.TopSites"])
+        navigator.openURL(website1["url"]!, waitForLoading: true)
+        let pageActionMenuButton = app.buttons["TabLocationView.pageOptionsButton"]
+        waitForExistence(pageActionMenuButton)
```

Comment:
Looking at the results in [BB](https://dashboard.buddybuild.com/apps/57bf25c0f096bc01001e21e0/build/5bd9b7b639ab730001e86cf7/test/1d802bf0-6d12-4112-a697-4361c4448438/2ef65ec4-0262-4f68-a854-a58fbe02e1cb) you may need to add here a timeout...

`waitForExistence(pageActionMenuButton, timeout: 5)`